src/app/qtsingleapplication/qtlocalpeer.cpp

```diff
@@ -119,7 +119,7 @@ bool QtLocalPeer::isClient()
         return true;
 
     bool res = server->listen(socketName);
-#if defined(Q_OS_UNIX) && (QT_VERSION >= QT_VERSION_CHECK(4,5,0))
```

Comment:
This is a 3rd party file. I don't think we should touch this. In any case, distros that build against their own qtsinglepplication package won't benefit from this.